src/Geocodio.php

```diff
@@ -37,7 +41,23 @@ class Geocodio
         'country',
     ];
 
-    public function __construct(private readonly Client $client = new Client) {}
+    /**
+     * @var Current SDK version
+     */
+    const SDK_VERSION = '1.2.0';
```

Comment:
Can bump this to 2.0.0 before merging in

Reply:
Yeah! I was going to update it before tagging but I'll do it here!